Lykke.Service.LykkeService/client/Lykke.Service.LykkeService.Client/LykkeServiceClient.cs

```diff
@@ -4,12 +4,18 @@ namespace Lykke.Service.LykkeService.Client
 {
     public class LykkeServiceClient : ILykkeServiceClient
     {
-        //public IControllerApi Controller { get; }
-        
+        //In case of several api controllers in your app - uncomment and use multiple APIs code below.
+        private ILykkeServiceClient _api;
```

Comment:
this must be a public property like Controller above, so client will use them like: _client.Controller.SomeMethodAsync()

Reply:
No, all client operations must be available via ILykkeServiceClient implementation.
If ILykkeServiceClient has Controller Property then it must be added separately and pass a call to _api fields.
In some cases LykkeServiceClient would be just a facade - passing calls from ILykkeServiceClient to seprate api subset via _api field.